tox.ini

```diff
@@ -1,32 +1,90 @@
 [tox]
 skipsdist = True
 minversion = 2.7.0
-envlist = {py27,py36}, linting, mcpserver, checkformigrations
+envlist =
```

Comment:
From what I could find, the tox documentation says that `envlist` should consist of comma-separated values. It's working as written here for me, but I'm a bit confused about why.

I'm curious too on your thoughts about dropping {py36} from the envlist. Is the idea that we would add it back later when we have reason to think the tests will pass?

Reply:
>From what I could find, the tox documentation says that `envlist` should consist of comma-separated values. It's working as written here for me, but I'm a bit confused about why.

Oh yeah, good point! I started with a single line of comma separated values but as you can imagine that gets really long which makes tracking diffs a bit difficult. Then I tried this comma separated variation:

```
envlist = archivematica-common,
          dashboard,
          mcpserver,
          mcpclient,
          mcpclient-ensure-no-mutable-globals,
          storage-service,
          checkformigrations-dashboard,
          checkformigrations-storage-service,
          linting
```

similarly to [how the tox project itself does](https://github.com/tox-dev/tox/blob/555f3f13b18da1470b6518be654e3e4e2fdec654/tox.ini#L2-L12), but didn't like the uneven indenting with the rest of the file and that final line with no trailing comma which carries the same diff issue.

So I used the same [approach of the pytest project](https://github.com/pytest-dev/pytest/blob/92959fe236e100b963a23570936c05136a3f75d2/tox.ini#L5-L17) which lead me to learn that [`tox` has been parsing the `envlist` as a multiline value first](https://github.com/tox-dev/tox/commit/1ee2e044fa06f95dd18969beafcbddc95b580eb0) for many years. And [it still does](https://github.com/tox-dev/tox/blob/7dbed25b63a0892bbc9e743f96d118ab237d6d82/src/tox/config/__init__.py#L1521).

>I'm curious too on your thoughts about dropping {py36} from the envlist. Is the idea that we would add it back later when we have reason to think the tests will pass?

Yes. When we ported the CI from Travis to GitHub Actions we considered that having broken environments had no much value. I think once we are ready to start porting to Python 3 we can use [generative envlists](https://tox.readthedocs.io/en/3.21.3/config.html#generative-envlist) to port each component separately if needed (e.g. switching to `{py27,py36}-mcpserver`, etc) if we handle the Python 3 requirements separately.

Reply:
> So I used the same approach of the pytest project which lead me to learn that tox has been parsing the envlist as a multiline value first for many years. And it still does.

Interesting! Good to know, and thanks for the explanation!

> When we ported the CI from Travis to GitHub Actions we considered that having broken environments had no much value. I think once we are ready to start porting to Python 3 we can use generative envlists to port each component separately if needed (e.g. switching to {py27,py36}-mcpserver, etc) if we handle the Python 3 requirements separately.

Cool, that sounds very sensible :)